tensorflow/python/BUILD

```diff
@@ -2313,6 +2313,7 @@ cuda_py_test(
         ":math_ops",
         "//third_party/py/numpy",
     ],
+    tags = ["no_windows_gpu"],
```

Comment:
I don't see any of the tests under `tensorflow/python/ops` running in cmake build, maybe we missed them?
But all these tests used to pass in GPU build, we definitely need to investigate the reasons.
Filed issue to track this https://github.com/tensorflow/tensorflow/issues/11962